### ISSUE TITLE
Keep Training but no output 

### ISSUE THREAD

**Issue by Lyt859165290** [CuriousAI/mean-teacher#16]
I used the suggested command 'python main.py \
    --dataset cifar10 \
    --labels data-local/labels/cifar10/1000_balanced_labels/00.txt \
    --arch cifar_shakeshake26 \
    --consistency 100.0 \
    --consistency-rampup 5 \
    --labeled-batch-size 62 \
    --epochs 180 \
    --lr-rampdown-epochs 210

I'm using ubuntu 18.04 python3.6, pytorch 0.3.0. numpy 1.14.2, and cuda8.0.and 2 gtx1080Ti
When I run the main.py, it can start training but there is no output information(epochs ,accuracy) during the training process (in a few hours), the only outputs are like this:

INFO:main:=> creating model 'cifar_shakeshake26'
INFO:main:=> creating EMA model 'cifar_shakeshake26'
INFO:main:
List of model parameters:
=========================
module.conv1.weight                            16 * 3 * 3 * 3 =         432
module.layer1.0.conv_a1.weight                96 * 16 * 3 * 3 =      13,824
module.layer1.0.bn_a1.weight                               96 =          96
module.layer1.0.bn_a1.bias                                 96 =          96
.....
module.fc2.weight                                    10 * 384 =       3,840
module.fc2.bias                                            10 =          10
===========================================================================
all parameters                                   sum of above =  26,197,316

I have checked the results folder and there is no checkpoint file in it.

**Comment by CheukNgai** [comment]
Hi, I got the same hang up issue when training with pytorch 0.3.1 and torchvision  0.2.0 under 4 gtx1080Ti. Could anybody tell me how to fix it?
